pkg/operator/podsecurityreadinesscontroller/conditions.go

```diff
@@ -14,12 +14,19 @@ import (
 )
 
 const (
-	PodSecurityCustomerType       = "PodSecurityCustomerEvaluationConditionsDetected"
-	PodSecurityOpenshiftType      = "PodSecurityOpenshiftEvaluationConditionsDetected"
-	PodSecurityRunLevelZeroType   = "PodSecurityRunLevelZeroEvaluationConditionsDetected"
-	PodSecurityDisabledSyncerType = "PodSecurityDisabledSyncerEvaluationConditionsDetected"
+	PodSecurityCustomerViolationType          = "PodSecurityCustomerEvaluationViolationConditionsDetected"
```

Comment:
Is there a good reason, why we change the name of that? It would mean that we need to change the metrics we are looking for in Thanos.

Reply:
I've consolidated down to 1 inconclusive type and put these names back the way they were to keep things simpler.